bots/pr/src/main/java/org/openjdk/skara/bots/pr/LimitedCensusInstance.java

```diff
@@ -73,7 +73,8 @@ private static Optional<JCheckConfiguration> configuration(HostedRepository remo
         try {
             conf = Optional.of(Arrays.stream(remoteRepo.fileContents(name, ref).split("\n")).toList());
         } catch (UncheckedRestException e) {
-            if (e.getStatusCode() != 404) {
+            // Throw the exception if the error is not exactly "File not found"
+            if (e.getStatusCode() != 404 || e.getBody().contains("Commit Not Found") || e.getBody().contains("No commit found")) {
```

Comment:
We should check for precisely "File Not Found" or "Not found".

```suggestion
            if (e.getStatusCode() != 404 || (!e.getBody().equals("File Not Found") && !e.getBody().equals("Not found"))) {
```

Reply:
Yes, it will be better. Thanks!